src/Model/Profile.php

```diff
@@ -1235,4 +1235,75 @@ public static function stripQueryParam($s, $param)
 	{
 		return preg_replace('/[\?&]' . $param . '=(.*?)(&|$)/ism', '$2', $s);
 	}
+
+	/**
+	 * search for Profiles
+	 *
+	 * @param int  $start
+	 * @param int  $count
+	 * @param null $search
+	 *
+	 * @return array [ 'total' => 123, 'entries' => [...] ];
+	 *
+	 * @throws \Exception
+	 */
+	public static function searchProfiles($start = 0, $count = 100, $search = null)
+	{
+		if ($search) {
+			$search = DBA::escape($search);
+
+			$sql_extra = " AND ((`profile`.`name` LIKE '%$search%') OR
```

Comment:
It's too late for this PR, but for future PR I strongly recommend to clearly separate SQL commands from parameters. This is needed for eventually being able to use PostgreSQL as well - and is much safer since this should avoid any SQL injections.

Reply:
How would you have made this query? I'll create a followup PR to add your suggestions

Reply:
You have to replace the ```$search``` with ```?``` in the query and have to add the value in the parameter array of the query (without escaping it). Just be sure that the numbers of parameters in query and parameter array do match. (otherwise there will be some error log by DBA.php).